### ISSUE TITLE
Bundles Rendered Inline on Production instead of with RenderScripts()

### ISSUE THREAD

**Issue by jwynveen** [andrewdavey/cassette#246]
I have this all working in my local development, but once it's uploaded to my dev or production server, resources are being rendered in place instead of being rendered where Bundles.RenderScripts() is called.

What I have is an MVC partial view that contains these references at the top of it:

```
<% Bundles.Reference(Url.Css("jquery.ptags.default.css", false)); %>
<% Bundles.Reference("jquery-ui-css"); %>
<% Bundles.Reference(Url.Javascript("TagControl.js", false)); %>
```

The TagControl.js also contains some references:

```
/// <reference path="~/jquery-ui" />
/// <reference path="jquery.ptags.js" />
```

My RenderScripts() call is at the bottom of my master page with all of my other scripts. When I load the page that has that partial on it from localhost, the scripts get rendered at the bottom where they should. Once I push the same code up to my dev/production servers, the scripts are rendered with the rest of the code in that partial. That causes a problem since it's trying to use jQuery before it's been loaded.

I tried copying my web.config from the dev server and ran it locally, but I couldn't reproduce it. What could cause this to happen?

**Comment by andrewdavey** [comment]
That's very weird! If you disable Cassette's HTML output rewriter do you get different behavior?

Web.config:

``` xml
<configuration>
  <configSections>
    <section name="cassette" type="Cassette.Configuration.CassetteConfigurationSection, Cassette"/>
  </configSections>
  <cassette rewriteHtml="false"/>
</configuration>
```

**Comment by jwynveen** [comment]
Sorry, it turns out this was actually an issue on my side. I was using that partial view inside an Area, trying to reference the partial inside ~/Views/Shared. However, on my servers, there was still an old version of the partial in ~/Areas/Admin/Views/Shared, so it was pulling that old partial that had the scripts inline. Everything was fixed once I deleted that extra file and it started referencing the correct partial.
